README.md

```diff
@@ -65,6 +63,10 @@ $ ./gradlew clean build bintrayUpload -PbintrayUser=BINTRAY_USERNAME -PbintrayKe
 
 More info on the available properties and other usages in the [Github Wiki](https://github.com/novoda/bintray-release/wiki).
 
+## Gradle compatibility
+| bintray-release version | Required Gradle version for | Notes                                        |
```

Comment:
> Required Gradle version for

I don't really understand this? Can you explain what it is saying sorry.

Ah maybe I get it. Would a better title be:

| Gradle (_wrapper_) version |
| ------ |
| for Java: 4.0 |
| for Android: 4.1 |

Reply:
What about this?

| bintray-release version | Java projects | Android projects | 
| --- | --- | --- | 
| 0.8.0 | Gradle 4.0+ | Gradle 4.1+ | 

**Notes: ** Currently Gradle 4.5 doesn't work with Android libraries

Reply:
yes 💯

Reply:
should we also write what version of the android gradle plugin is needed?

Reply:
I would not do that.
If we start using that "pattern" we have to
* always make sure to take a look which android gradle plugin requires which gradle version (which isn't our responsibility)
* more or less copy the information from the android documentation in our readme

This is just maintenance overhead.

Another example:
Imagine a user have a project with the andorid gradle plugin 2.3.3 and are interested in this plugin.
The see that our plugin requires 4.1 gradle. So he updated it's gradle version to 4.1. and everything works (should works 😄 )
But if we say "android gradle plugin 3.x" he assume that it only works if we update it's grade plugin. Which isn't needed for the bintray-release plugin.